backend/tests/unit/test_api.py

```
@@ -0,0 +1,557 @@
+import pytest
+import json
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)